visual_excuses/main.py

```diff
@@ -161,6 +166,12 @@ def create_visual_excuses(data, team_choice='', age=0):
             # Don't display the node if it's younger than the --age flag
             if age and item['age'] < age:
                 continue
+            if item['update-excuse-bugs']:
+                bugs = ", ".join([f'<a href="https://pad.lv/{bug}">{bug}</a>' for bug in item['update-excuse-bugs']])
```

Comment:
You should link to launchpad.net directly. Going through an other additional service is an additional point of failure and it can easily be avoided here.

Reply:
agree I think it would be better to use launchpad.net/bugs/{bug} I can update this.